pkg/loop/internal/relayer/pluginprovider/chainreader/chain_components_test.go

```diff
@@ -65,20 +65,21 @@ func TestVersionedBytesFunctions(t *testing.T) {
 	})
 }
 
-func TestChainReaderInterfaceTests(t *testing.T) {
+func TestChainComponentsInterfaceTests(t *testing.T) {
```

Comment:
I assume that CW will still need its own set of tests, in that case it would make more sense to keep chainReader everywhere instead of chain components